config/initializers/bgs.rb

```diff
@@ -4,7 +4,7 @@
 
 LighthouseBGS.configure do |config|
   config.application = Settings.bgs.application
-  config.client_ip = Socket.ip_address_list.detect(&:ipv4_private?).ip_address
+  config.client_ip = Socket.ip_address_list.detect(&:ipv4_private?)&.ip_address
```

Comment:
Does this apply to this PR? Maybe a rubocop thing?

Reply:
no, it was just something I caught in testing with vets-api while not having an internet connection. I could remove it from this PR and do a separate one if it's a problem.

Reply:
I'm not sure letting it be `nil` is helpful. Pretty much an edge case imho when you don't have an internet connection -- no different than changing it to a one-off URL.

Maybe a default would be a better choice here (eg `|| 127.0.0.1`)

Reply:
k, maybe i'll pull it out right now and can be addressed in a different PR.

Reply:
pulled in most recent commit. @johnpaulashenfelter